tests/profiling/settings.js

```diff
@@ -7,6 +8,7 @@ module.exports = {
     setup: {
         resolver: getAssets,
         uniqueParams: ['language'],
+        store: redisStore('my-namespace', { host: '0.0.0.0', port: 6379 }),
```

Comment:
It would be great to make running with a redis store configurable or commented out by default so the benchmark can be run by default without a local redis instance.

Reply:
ah, right my bad, I'll comment it out